core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ChangeLoggable.java

```diff
@@ -11,28 +9,10 @@
  * Marker interface to indicate an implementing class can generate change logs.
  *
  */
-public interface ChangeLoggable {
+public interface ChangeLoggable extends Timestamped {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)